install_files/ansible-base/roles/postfix/templates/sasl_passwd

```diff
@@ -1 +1 @@
-[{{ smtp_relay }}]:{{ smtp_relay_port }}   {{ sasl_username }}@{{ sasl_domain }}:{{ sasl_password }}
+[{{ smtp_relay }}]:{{ smtp_relay_port }}   {{ sasl_username }}{% if sasl_domain %}@{{ sasl_domain }}{% endif %}:{{ sasl_password }}
```

Comment:
Ideally we would have molecule tests checking both cases (empty/not empty) work but we don't have that yet. However it looks fine visually :-)